CHANGELOG.md

```diff
@@ -1,9 +1,53 @@
 Changes by Version
 ==================
 
-1.21.0 (unreleased)
+1.22.0 (unreleased)
 -------------------
 
+1.21.0 (2020-11-13)
+-------------------
+
+### Backend Changes
+
+#### Breaking Changes
+
+* Remove deprecated es.max-num-spans ([#2482](https://github.com/jaegertracing/jaeger/pull/2482), [@BernardTolosajr](https://github.com/BernardTolosajr))
```

Comment:
This one isn't merged. I would skip it for this release and do a larger deprecation in the next release (we have accumulated quite a few deprecated flags).